saga-core/src/main/java/org/apache/servicecomb/saga/core/Fallback.java

```diff
@@ -34,6 +34,11 @@ public SagaResponse send(String address) {
     public SagaResponse send(String address, SagaResponse response) {
       return send(address);
     }
+
+    @Override
+    public int retries() {
+      return 0;
```

Comment:
why it return 0 here ? it means to retry infinite ?

Reply:
No, Fallback does not retry currently.
Because I abstracted the retires method from Compensation to Operation, Fallback needs to implement this method temporarily.
Considering the default number of retries if Fallback needs to retry in the future.